EvidenceApi/V1/Gateways/NotifyGateway.cs

```diff
@@ -12,5 +13,44 @@ public NotifyGateway(INotificationClient client)
         {
             _client = client;
         }
+
+        public void SendNotification(DeliveryMethod deliveryMethod, CommunicationReason reason, Resident resident)
+        {
+            var result = Deliver(deliveryMethod, reason, resident);
+        }
+
+        private NotificationResponse Deliver(DeliveryMethod deliveryMethod, CommunicationReason reason, Resident resident)
+        {
+            return deliveryMethod switch
+            {
+                DeliveryMethod.Email => _client.SendEmail(resident.Email, GetEmailTemplateId(reason), null, null, null),
+                DeliveryMethod.Sms => _client.SendSms(resident.PhoneNumber, GetSmsTemplateId(reason), null, null, null),
```

Comment:
should we check if `SendEmail` and `SendSms` are overloaded so that we don't pass `null` as arguments unless we really have to?